src/Modules/ModParser.lua

```diff
@@ -1326,6 +1326,22 @@ local function triggerExtraSkill(name, level, noSupports)
 		}
 	end
 end
+local function supportedByEffects(name, level)
+	local skillId = gemIdLookup[name] or gemIdLookup[name:gsub("^increased ","")]
+	if skillId then
+		local gemId = data.gemForBaseName[data.skills[skillId].name .. " Support"]
+		if gemId then
```

Comment:
These can be in-lined.

```suggestion
	if gemIdLookup[name] or gemIdLookup[name:gsub("^increased ","")] then
		if data.gemForBaseName[data.skills[skillId].name .. " Support"] then
```

Reply:
@Wires77 the variable `skillId` is used on line 1331. You deleted the variable when in-lining.